frontend/src/scenes/sessions/sessionsTableLogic.ts

```diff
@@ -207,7 +194,8 @@ export const sessionsTableLogic = kea<
 
             if (
                 JSON.stringify(params.properties || []) !== JSON.stringify(values.properties) ||
-                JSON.stringify(params.duration || {}) !== JSON.stringify(values.duration) ||
+                JSON.stringify(params.duration || {}) !== JSON.stringify(values.duration || {}) ||
+                !values.selectedDate ||
```

Comment:
Context on this?

Reply:
In case someone else who needs context, from what we discussed in our call, this fixes the bug where loading the sessions page for the first time (`/sessions`) doesn't set a default date and therefore causes issues with the previous/next dates.